photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameParams.scala

```diff
@@ -148,7 +153,7 @@ class GameParams extends FeatureParams with PalDBIndexMapParams with EvaluatorPa
   var modelOutputMode: ModelOutputMode = ALL
 
   /**
-   * Number of output files to write for each random effect model.
+   * Number of output files to write for each random effect model
```

Comment:
Here's an informal system I've arrived at over the years:

- If you have only a dependent clause or just one sentence, no period
- If you have multiple sentences, use periods

I think the reason for this is that you definitely don't want to use a period for just a dependent clause, and when there's just one sentence it's a little superfluous. I'm not married to this though, and if you guys want to put a period for every sentence, I'm okay with that.